pkg/identityserver/gateway_registry.go

```diff
@@ -464,6 +467,7 @@ func (is *IdentityServer) updateGateway(ctx context.Context, req *ttnpb.UpdateGa
 			return nil, err
 		} else if req.LBSLNSSecret != nil {
 			value := req.LBSLNSSecret.Value
+			ptLBSLNSSecret = req.LBSLNSSecret.Value
```

Comment:
Shouldn't this just replace the `value` above?

Same question for the other secrets below.

Reply:
No because the `value` gets overwritten by the encrypted secret if an encryption key is defined.